spec/definitions.yaml

```diff
@@ -788,3 +795,268 @@ definitions:
       auth_token:
         type: string
         description: The newly created API token
+
+
+  # List of node pools
+  V5GetNodePoolsResponse:
+    type: array
+    items:
+      $ref: '#/definitions/V5GetNodePoolResponse'
+
+  # Add node pool
+  V5AddNodePoolRequest:
+    description: |
+      Request body structure for creating a node pool
+    type: object
+    properties:
```

Comment:
Similary do we know which fields are required when creating a nodepool creation request?